aws/resource_aws_cloudhsm2_cluster_test.go

```diff
@@ -84,6 +84,8 @@ func testSweepCloudhsmv2Clusters(region string) error {
 }
 
 func TestAccAWSCloudHsmV2Cluster_basic(t *testing.T) {
+	resourceName := "aws_cloudhsm_v2_cluster.test"
```

Comment:
This is currently failing with:

```
--- FAIL: TestAccAWSCloudHsmV2Cluster_basic (258.74s)
    testing.go:669: Step 0 error: Check failed: Check 1/6 error: Not found: aws_cloudhsm_v2_cluster.test
```

Will switch it to the existing name for now:

```suggestion
	resourceName := "aws_cloudhsm_v2_cluster.cluster"
```